app/src/main/java/com/zgamelogic/ghosty/data/Ghost.java

```diff
@@ -29,33 +33,27 @@ public Ghost(String name, String description, int id, LinkedList<String> evidenc
         this.name = name;
         this.description = description;
         this.id = id;
-        this.evidence = evidence;
+        this.evidence = new HashSet<String>(evidence);
```

Comment:
String can be removed here, java can infer the collection type based on the declaration above and the collection type being passed in.
```suggestion
        this.evidence = new HashSet<>(evidence);
```